### PR TITLE
Added Workers Defense Project, Asian Family Support Services of Austin.

### PR DESCRIPTION
**Pull request by mattThousand**
`Workers Defense Project` - empowers low-income workers to achieve fair employment through education, direct services
`Lawyers’ Committee for Civil Rights Under Law` - civil rights organization providing advocacy and legal representation for ethnic minorities.
`Southern Christian Leadership Conference`- national African-American civil rights organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Please find those that resonate with you, donate time, money, or both.
 - [NAACP LDF (](http://www.naacpldf.org/)people of color)
 - [Mexican American Legal Defense and Education Fund](http://www.maldef.org/)
 - [Muslim Justice League](https://www.muslimjusticeleague.org)
+- [Workers Defense Project](http://www.workersdefense.org/)
+- [Asian Family Support Services of Austin](http://www.saheli-austin.org/)
```
Comment (graiz) [inline README.md]: Trying to keep it on the national level when possible. Is this a chapter of a national org?

Comment (mattThousand) [reply README.md]: Good call, it is very local. WDP is regional (SouthWest), SCLC and LCCRUL are national.

File: README.md [hunk 2]
```diff
@@ -45,7 +45,8 @@ Please find those that resonate with you, donate time, money, or both.
 - [Mexican American Legal Defense and Education Fund](http://www.maldef.org/)
 - [Muslim Justice League](https://www.muslimjusticeleague.org)
 - [Workers Defense Project](http://www.workersdefense.org/)
-- [Asian Family Support Services of Austin](http://www.saheli-austin.org/)
+- [Lawyers' Committee for Civil Rights Under Law](https://lawyerscommittee.org/)
+- [Southern Christian Leadership Conference](http://nationalsclc.org/)
```
Comment (graiz) [inline README.md]: I like the lawyerscomittee addition. I'm not as certain about the SCLC org, it seems to have a larger religious undertone.  My concern is that some religious organizations may limit the civil liberties if they don't agree with the religious view.  Can you vouch that they done?

Comment (mattThousand) [reply README.md]: the SCLC organizes seminars/workshops in communities where human trafficking occurs to help them address root causes, but I think historically they've been associated with African American civil rights causes.

Their actual impact seems a little abstract to me ("raising awareness" and such), which is certainly less appealing than the other organizations you identified that provide concrete services for people. Also, you're right, the religious overtone makes it exclusive. Removing

**Pull request merged**